dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="main.js"></script></head>
+<body>
+    <div class="header">
+        <h1>Leaderboard</h1>
+    </div>
+
+    <div class="score-container">
+    
+    </div>
+    
```

Comment:
- [ ] Please, make sure that index.html should only have a root element. 🐶

Reply:
This issue is well addressed. Great work! :clap: :clap: